scanner/src/main/java/no/nordicsemi/android/blinky/scanner/ScannerDestination.kt

```diff
@@ -1,25 +1,21 @@
 package no.nordicsemi.android.blinky.scanner
 
-import androidx.core.os.bundleOf
+import androidx.hilt.navigation.compose.hiltViewModel
+import no.nordicsemi.android.blinky.control.Blinky
+import no.nordicsemi.android.blinky.control.BlinkyDevice
```

Comment:
Is there no way then to keep blinky modules and scanner module completely separated and independent? And then have only the app module be the one that performs the navigation / transport of data as parameters from one module to the other.

Reply:
That's a very good question. Let's discuss it in 3 minutes.